gammapy/maps/geom.py

```diff
@@ -1296,6 +1296,17 @@ def upsample(self, factor):
         """
         pass
 
+    @abc.abstractmethod
+    def solid_angle(self):
+        """
```

Comment:
Adjust docstring here as well. Which docstring ends up in the derived classes in the HTML docs?